pkg/k8sutil/operator.go

```diff
@@ -37,6 +39,22 @@ func RestartDeployment(ctx *Context, namespace, deploymentName string) {
 	fmt.Printf("deployment.apps/%s restarted\n", deploymentName)
 }
 
+func GetOperatorPod(ctx *Context, operatorNamespace string) (corev1.Pod, error) {
```

Comment:
If we're waiting, it's helpful to have that in the method name
```suggestion
func WaitForOperatorPod(ctx *Context, operatorNamespace string) (corev1.Pod, error) {
```